tests/sals/kubernetes/test_kubernetes.py

```diff
@@ -0,0 +1,175 @@
+        self.kube_manager.install_chart(release_name, release_chart)
+        self.info("Check if chart installed")
+        is_installed = False
+        for release in self.kube_manager.list_deployed_releases():
+            if release["name"] == release_name and release["status"] == "deployed":
+                is_installed = True
+                break
+        self.assertTrue(is_installed)
+
+    def test_05_list_deployed_charts(self):
+        """Test case for listing all deployed charts.
+
+        **Test Scenario**
+
+        - Deploy VDC
+        - List all deployed charts
+        - Check that all deployed charts are listed.
+        """
+        self.info("list all deployed charts")
+        all_deployed_charts = self.kube_manager.list_deployed_releases()
+        is_listed = False
+        self.info("Check that deployed charts listed")
+        if not (all_deployed_charts is None):
+            is_listed = True
+        self.assertTrue(is_listed)
+
+    def test_06_delete_deployed_chart(self):
+        """Test case for delete deployed chart.
+
+        **Test Scenario**
+
+        - Deploy VDC
+        - Deploy ETCD Chart
+        - Delete Deployed Chart
+        - List all deployed charts
+        - Check that deleted chart not in deployed charts.
+        """
+        self.info("Deploy ETCD chart")
+        release_name = "testdeletechart"
+        release_chart = "bitnami/etcd"
+        self.kube_manager.install_chart(release_name, release_chart)
+        self.info("Delete etcd release that installed before")
+        self.kube_manager.delete_deployed_release(release_name)
+        self.info("Check if deployed chart deleted")
+        is_deleted = True
+        for release in self.kube_manager.list_deployed_releases():
+            if release["name"] == release_name:
+                is_deleted = False
+                break
+        self.assertTrue(is_deleted)
+
+    def test_07_is_helm_installed(self):
+        """Test case to check if helm installed.
+
+        **Test Scenario**
+
+        - Deploy VDC
+        - Check if helm installed.
+        """
+        self.info("Check if helm installed")
+        is_helm_installed = j.sals.kubernetes.manager.is_helm_installed()
+        self.assertTrue(is_helm_installed)
+
+    def test_08_execute_native_cmd(self):
+        """Test case for executing a native kubernetes commands.
+
+        **Test Scenario**
+
+        - Deploy VDC
+        - Excute Kubernetes Command
+        - Check that the command executed correctly.
+        """
+
+        self.info("Execute kubectl get nodes")
+        cmd_out = self.kube_manager.execute_native_cmd("kubectl get nodes")
+        self.info("Check if command executed correctly")
+        # Check is True if there is one master node and at least 2 Ready nodes
+        is_executed = False
+        if cmd_out.count("master") == 1 and cmd_out.count("Ready") >= 2:
+            is_executed = True
+        self.assertTrue(is_executed)
```

Comment:
in case of the master is only deployed, this test would fail
let's check for only one node is ready or more
```suggestion
        self.assertEqual(cmd_out.count("master"), 1)
        self.assertGreaterEqual(cmd_out.count("Ready"), 1)
```